test/tools/libtesteth/boostTest.cpp

```diff
@@ -148,7 +148,8 @@ int main(int argc, const char* argv[])
     }
 
     // Print suggestions of a test case if test suite not found
-    if (!sMinusTArg.empty() && !dev::test::inArray(c_allTestNames, sMinusTArg))
+    if (!sMinusTArg.empty() && !dev::test::inArray(c_allTestNames, sMinusTArg) &&
+        sMinusTArg != "customTestSuite")
```

Comment:
what does this change do?

Reply:
disables name suggestion check for customTestSuite which is created dynamicly